src/System.Private.CoreLib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -227,8 +234,9 @@ public void Clear()
             }
         }
 
-        public bool ContainsKey(TKey key)
-            => FindEntry(key) >= 0;
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Is this really needed? It should not be needed, especially once the method uses Unsafe.IsNull.